lively.morphic/world.js

```diff
@@ -47,9 +49,19 @@ export class World extends Morph {
           const { showsUserFlap } = resource(document.location.href).query();
           if (typeof showsUserFlap !== 'undefined') bool = showsUserFlap;
           this.setProperty('showsUserFlap', bool);
-          this.whenReady()
-            .then(() => System.import('lively.user/morphic/user-ui.js'))
-            .then(userUI => userUI.UserUI[bool ? 'showUserFlap' : 'hideUserFlap'](this));
+          System.import('lively.user/morphic/user-ui.js')
+            .then(userUI => userUI.UserUI[bool ? 'showUserFlap' : 'hideUserFlap'](this)
+              .then(() => { if (bool) this.enableComments = true; }));
```

Comment:
I don't understand what this does here

Reply:
It uses the Promise returned by System.import() and creates the userFlap ("showUserFlap") once it was imported. This command returns a Promise, which we use now to enable our Comments (and thus create our browser) once the Topbar was loaded and created